suite-native/module-onboarding/src/components/SecuritySealDescription.tsx

```diff
@@ -0,0 +1,96 @@
+import { useState } from 'react';
+import { useSelector } from 'react-redux';
+
+import { selectDeviceModel } from '@suite-common/wallet-core';
+import { BottomSheet, Box, Button, Text, VStack } from '@suite-native/atoms';
+import { Icon } from '@suite-native/icons';
+import { Translation } from '@suite-native/intl';
+import { Link, useOpenLink } from '@suite-native/link';
+import { DeviceModelInternal } from '@trezor/connect';
+import { HELP_CENTER_PACKAGING_T3B1_URL, HELP_CENTER_PACKAGING_T3T1_URL } from '@trezor/urls';
+
+import { SecuritySealImages } from './SecuritySealImages';
+
+export const SecuritySealDescription = () => {
+    const openLink = useOpenLink();
+    const [isBottomSheetVisible, setIsBottomSheetVisible] = useState(false);
+
+    const openBottomSheet = () => setIsBottomSheetVisible(true);
+    const closeBottomSheet = () => setIsBottomSheetVisible(false);
+
+    const deviceModel = useSelector(selectDeviceModel);
+
+    const isTrezorSafe5 = deviceModel === DeviceModelInternal.T3T1;
+
+    const knowledgeBaseLink = isTrezorSafe5
```

Comment:
Does make sense to just inline this?
```suggestion
    const knowledgeBaseLink = deviceModel === DeviceModelInternal.T3T1
```

Reply:
[9cb8378](https://github.com/trezor/trezor-suite/pull/16883/commits/9cb8378fbdc1a83ca45c972789f5100c40de8415)